node/package.json

```diff
@@ -32,7 +32,7 @@
         "compile-protobuf-files": "cd src && pbjs -t static-module -o ProtobufMessage.js ../../glide-core/src/protobuf/*.proto && pbts -o ProtobufMessage.d.ts ProtobufMessage.js",
         "clean": "rm -rf build-ts rust-client/target docs glide-logs rust-client/glide-rs.*.node rust-client/index.* src/ProtobufMessage.*",
         "fix-protobuf-file": "replace 'this\\.encode\\(message, writer\\)\\.ldelim' 'this.encode(message, writer && writer.len ? writer.fork() : writer).ldelim' src/ProtobufMessage.js",
-        "test": "npm run build-test-utils && jest --verbose --testPathIgnorePatterns='ServerModules'",
+        "test": "npm run build-test-utils && jest --verbose --runInBand --testPathIgnorePatterns='ServerModules'",
```

Comment:
Here it's better to avoid, just for modules. It doubles the time, and the CLI endpoint is not in use.

Reply:
Talking about the inband.

Reply:
In general, i think our goal is to avoid testing one by one, and utilizing the whole cores of the machine.

Reply:
Ok, I'll add another task and update developer guide. This thing is essential for debugging.

Reply:
Keep in mind that we have a number of client blocking tests and if they are getting executed in parallel - they could fail.

Reply:
Our blocking are very short, isn't it?
If we will see some cases we will need maybe to think on some kind of fence. Anyhow theres some replaning to do around the tests.
If we cut test amount by half (a dream) we can afford more games, and if we have more runner with jest theres technic of dividing tests to x and then you can run it in a matrix.
But currently our issue us runners so its not helpful.

Reply:
Created #2744